packages/extension-chakra-storefront/README.md

```diff
@@ -0,0 +1,156 @@
+/    \  \/|  |  \\__  \ |  |/ /\_  __ \__  \  \_____  \\   __\/  _ \_  __ \_/ __ \   __\\_  __ \/  _ \ /    \   __\
+\     \___|   Y  \/ __ \|    <  |  | \// __ \_/        \|  | (  <_> )  | \/\  ___/|  |   |  | \(  <_> )   |  \  |  
+ \______  /___|  (____  /__|_ \ |__|  (____  /_______  /|__|  \____/|__|    \___  >__|   |__|   \____/|___|  /__|  
+        \/     \/     \/     \/            \/        \/                         \/                         \/      
+
+
+# Description
+
+The Chakra Storefront Application Extension is a fully featured composible storefront using Chakra UI for its primary component
+library, along with other first and third party librarys like ReactIntl to handle localization and CommerceSDKReact to provide a
+React Hook library for accessing your Commerce Instance. This is a great place to start your storefront implementation if you are 
+looking to get off to a quick start!! 
+
+Please refer to the list of features that this extesnion supports below. Also, noted below is the public API of files that you can 
+override if you choose to customize you application that way. 
+
+# Features
+
+The Chakra Storefront is a fully fledged storefront that includes the below features:
+
+- Localization support using ReactIntl.
+- Components provided by Chakra UI.
+- Implementations of the following pages:
+    - Home
+    - Product List
+    - Product Detail
+    - Cart
+    - Checkout
+    - Account
+    - Login
+    - Order Summary
+    - Wishlist
+
+# Overridable API
+
+Below is a list of all files this Application Extension allows to be overridden. This means that each of the below files can be overridden
+in the base project or any other configured appllication extension that is configured after it.
+
+Files:
+
+- /pages/account
+- /pages/cart
+- /pages/checkout
+- /pages/confirmation
+- /pages/home
+- /pages/login
+- /pages/registration
+- /pages/reset-password
+- /pages/login-redirect
+- /pages/product-detail
+- /pages/product-list
+- /pages/wishlist
+
+# Peer Dependancies
+
+PWA-Kit Application Extensions are NPM packages at their most simplest form, and as such you can define
+what peer dependencies are required when using it. Because this sample application extension provides
+UI via a new "Sample" page, it requires that the below dependencies are installed at a minimum. 
+
+Depending on what features your application extensions provides it's recommended you include any third-party
+packages as peer dependencies so that your base application doesn't end up having multiple versions of a 
+given package.
+
+"react": "^18.2.0",
+"react-dom": "^18.2.0"
+
+# Error Page
+
+This extension provides a error page that can be used as a replacement for the default provided by the PWA Kit
+React SDK. This page is NOT automatically wired up into the base application, its up to you to integrate it if you
+choose to use it. To accomplish this follow the below steps:
```

Comment:
I see that the `<Error />` component doesn't have access to static elements from the `extension-chakra-storefront`, like the logo and icons. Is this expected behavior?
<img width="1769" alt="Screenshot 2024-11-12 at 4 43 53 PM" src="https://github.com/user-attachments/assets/0f3ed2b8-c7c8-43e3-bdd2-c52cf25eee41">

How will users be able to integrate the Error page into the extension-chakra-storefront `_app` component so that the Header and Footer are displayed on the error page as they currently are?
https://pwa-kit.mobify-storefront.com/aaa

Reply:
I think this is a result of the hoc application order. But I'll have to check. 

As for the header footer stuff.. I think that would have to be something that is either modified in the error component itself to include them. I can make a follow up PR for this.

Reply:
I created [this](https://gus.lightning.force.com/lightning/r/a07EE000025XTUlYAO/view) ticket